src/pages/[platform]/build-a-backend/auth/multi-step-sign-in/index.mdx

```diff
@@ -1474,9 +2203,72 @@ func confirmSignIn(totpCode: String) -> AnyCancellable {
 
 </BlockSwitcher>
 
+
+## Confirm signin with Email MFA
```

Comment:
signin -> sign-in

Reply:
the prose following this heading should largely align with other platforms. The primary difference here is the "step" name

Reply:
i've [updated](https://github.com/aws-amplify/docs/pull/8023/commits/c61ad3e8f89e355217ad32b4349a1c9f02122695) `signin` to `sign-in` across all the headings for this page

Reply:
[and updated the language in these sections to align with other platforms](https://github.com/aws-amplify/docs/pull/8023/commits/97ec29135e7abb6608caa66ed5c7b4d49092c1be). lmk if this isn't want you intended